baqend/chromeUXReports.ts

```diff
@@ -1,38 +1,39 @@
 // Imports the Google Cloud client library.
-import {BigQuery} from '@google-cloud/bigquery';
-import {URL} from 'url';
-import credentials from './credentials';
-import {baqend, model} from "baqend";
+import {BigQuery} from '@google-cloud/bigquery'
```

Comment:
Der ganze Code sollte ja eigentlich in dem anderen MR sein. Einmal bitte mit rüber ziehen. Und darauf achten, dass wir im Backend code semicolons benutzen.